src/shared/prerendered-app/Intro/style.css

```diff
@@ -1,3 +1,11 @@
+/* Just the glyphs needed for the Squoosh logo */
+@font-face {
+  font-family: logofont;
+  font-weight: 700;
+  src: url(data:font/woff;base64,d09GMgABAAAAAASoAA4AAAAACOQAAARUAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGhYbgnYcgXAGYABkEQwKhHiEFgsUAAE2AiQDJAQgBYJaByAbeQfIxId/N/1zb0JJk7qwVQyoSyhM6qJf2zN3wnP5MyX/c6mabKKFAVwk4QDd+3f7Z41FoYvttryoVK6NsWls0wJjfeO38YjNiCJjtHr/1Q8CwAqDCAUBr2QweT2PEoyA1elzSuDRPDnUg8D2odZuRPU0jvQhDaxz5rMh0dmYGFMJs4nu1qE+eGjM/weALmNBwUEBOxTgRXxFMYpg//e8Ci0oLAD5Z/l7jALyz0eIk0p5wwIKMGmoUVDk+I2Voc2UQOpf+V8SHykzsc2YglhhpAxij4qYCCDLLiZGpIxaxGLgdEfA8VXagNHBPOEGN4kdxIwSYCOPFjwtuDxUnqS+IGdGuWtrAgUsQYZRHpDu7Qiw6ukb7cWFrWzAQoFsxEIZPaFS7IAQDQ4cvEx4DjEL+W8XY9w93bi6CVjaTB8XOUO3aIVhv7jXI3UoIVGDVOAYRkFZmv6lSiyebcjPN+BqWJ91Zp43fwUwPxAfFchViYiY6etaTAWH6kKIhE7SgEaAL0DMDnZmf6FCMTrQj0EMY1SW726Rv5Tvki/Jp/KJfCwf5XNoQXWWGi/LsNXQcgvxSnnJ8wFi43KUbaZAdno0JKEudnYudkHOvy6ZysqvkYTtt8jiu2VxJhNZfP99YVuKLLnyTX7rdSy8cT5WkrBgNPL1AkqEuyzOB+WZxXf5LSMoLzmnXHS9/9b7ZNHk7wAjFqQ3hMZy/nbFtQXbNAIRWP4jQ8kUVX7FG8L2Y4VH4/QvOjyltORefuthLJSWBnyUZCKMQBfeEbbfP3h5MvqPEuA12/1Or4a5yB8MYpzmv+DgZ3O9ntK5NjZdTEis7owRG/urcsYir+4JU2+cUdUY36zxbeoo1RsaBsJTkP7a8gPKCq64OCyytqmvKXVZE5mtjskqTmWW8Y3ttF43lZujm57W5+dN63XTObm6qWl9XsDErwGiVhMg/hoY+GugqNEE6n4NxH280sBFFITFZA/oMqby8zKmB3TZMfnqDC5tyrRUm/F7gKhWB4i/Bfj/FiCq1QHi7/643nboYs9I1K/XB/n/4eH+8COPf1aYl6yOya4KiL8ypq8/Qso6O+HoL9pfsL/okJObGJlwoSkkBZds6wfb4pYDYzxCEjIL09ll83QG11IUGgUAAJrLVRQvuOTOlNTbpvzOCcwPZllp+cGK/bUffKT+wRxg/trKyDwPAmVzJTtUmnnR/Ca2bGU0B5hvtjI2u3ZLmoci/IQP0Fz40ipk0Hb402H4k7/hScuRQrJj7rXEg+BR85QaWsDzADqVUNUxBLbRYp0JnCMELkwBnsiOEWgYqDGjYaHCzRoFPPDyyy3gg68goh8DmMQQOtGODoxAhShEIBKx9DgD+tGPdvSgFSrkIAcitEolEz3oIVeRDxy+4lvVSiuGMEbLtUDr97BiNKIPw8hCvzLeYjOefMW4mq5hUSguFWNZnL3D6EQ/+tLPa3ktEYgO08G8O9bbQgkWAA==)
```

Comment:
@jakearchibald This can be a bit smaller.

```suggestion
  src: url(data:font/woff;base64,d09GMgABAAAAAAR4AA4AAAAACOQAAAQiAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGhYbgnYcgXAGYABkEQwKhHiEFgsUAAE2AiQDJAQgBYJaByAbeQdRlDDKnwBfHGQqg+VFm0sR5RBxk8na4qIsy2f52/15xrN4+G+N330z+8UMl9PNLdJolEIlJEIySXgTq+Y7/3bZn8QzOXtlh3B1YAmhCjS75zrvWqkA/uthOQEFyP/nmOmqbCDRNsACGljZgCr+qkojQSe6N498vhMZyJ0f3YSOj87P5XSOpWuxmSwEObhs/ZIC9YAkJFVCoNZGNWIWLrZ4wN4DjVUlAHekKrcYsE5m1pQBmQLKUmALh3BC5C1OHKoqA3v5aQiEQlKtQrO1XnvttWSCAvdMk1RSfvoBgMlArAiBq1fIIuOpZKUYbJbnDSn/SApjlraMI9YqLtMOITPQ6RVCgoKceWVIBNMK/xztQoLXuh4DrX5Lt0QGlYgndl76oKdoElqEN6HzavWcbKhQI5tUqv2crEqd+pOna0uBSUOdleBAXZTgM1SkOHakXq1av6e7WqgsfyHXqjtZ1SOhSAfSQxWkKykAWKH/YWI3T7BIddU5V6QiFagCgCVr7HpwRsOojvzy/2vyxxgcFlch/qc5DGtFQka7LNCusBbQrFnGsA0KlKtUrbbcX1u+euXxs2dOnxKE/q/5vrVlqQnUgFbU4gbRCCjgm9uSVYpJnc3Nw1Om4ndAEBr2+c+iyPAi1Lni44XDUGckggyjTM+pBXhoDpg3clEUWFuD0xGwEbkCSIHg6Ag1W9YMJxqmR4Yii5E5azh+zxqAziOZfvji4RyqqlERCImB3wmiYZb/5Dwy/CB/n6d7HRDyeq6j6ncBr3cBNbQFikaiOpbJGjE8MJT/Q0BA44X7M9qY63o1j/qXQHhu3f9EsyszyywQJhdy1JnlIUsd80wJmTJw5HDK2kLKoawCr06fUUGXAPnuW/GBBLebzEzNKsuSdlOZWgpH5ZZGd4M16zRNVoumubntNpvWbLFqmkZs2IYvWDWNOoa+4HBnSU2l4jTPPXADjtcnMBxkjrZCo2gKcima7/1ajp2iSJA1hbtSFd+ai0IZc1+xmK/W//UNAy5sqjKX1LC+XMBjvu/dc/few09Om5jC0Yaw/FOcsnIGqjrasA2j3mLaYt5qsQqZAlMWUQKOb0qvzON14zh7iQKlUx7T/b9ZkQCfs57EAggkNzutLZ70TZJv1Uj+CE8uXw8APH9JWf8/9v+b+sf5hlAlgSDf+v8InwaBuLZBED9RTVrN5TNplUMpRJHyYVI1TPzSl/wkoQUEalxXa6fYPG/LMqpsFy2KGuwGbHbotJudVLvIzqa4xC4Mu8au0OvO+6g06LUFylVoVKVQvgI1DptlhpnmGrZEuXL5SuQ6bCULNZph85Qocdj6SKjmV+WqlqtKnVw5aDm+ukGmMtVUypXIQayJqwntpNIM2ZyHCXxrd1crVK7sPXYzzQwzzM7tjSaUlVdsBAAAAA==) 
```